syncto/views/collection.py

```diff
@@ -61,6 +61,10 @@ def collection_get(request):
     records = sync_client.get_records(collection_name, full=True,
                                       headers=headers, **params)
 
+    statsd = request.registry.statsd
```

Comment:
should we provide an helper for this? `statsd(request, "count", "syncclient.status_code.200")`